.circleci/continue_config.yml

```diff
@@ -57,13 +57,13 @@ jobs:
           name: Run linters and code style checks
           command: make py-style
       - run:
-          name: Run unit tests
-          command: make unit-test
+          name: Run integration tests
```

Comment:
why did you get rid of unit tests? is it moved somewhere?

Reply:
Unit tests still exists. See make file. CI/CD now runs integration which includes includes unit tests.